### PR TITLE
Portfolio-details-popup-window

### PR DESCRIPTION
**Pull request by FaranoshAmini**
in this project I implemented the following interactions:
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.

I also implemented popups for both mobile and desktop screen sizes:
- Mobile popup
- I followed all project requirements.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -264,6 +264,113 @@ section {
   list-style: none;
 }
 
+.pop-window {
+  height: 100vh;
+  width: 80%;
+  margin-left: 10%;
+  padding: 2%;
+  background-color: #fff;
+}
```
Comment (vechicin) [inline style.css:273]: Your design is not following the template design in either version.

Your design:

Screen Shot 2022-06-15 at 8 26 19

Screen Shot 2022-06-15 at 8 26 47

The template design:

Screen Shot 2022-06-15 at 8 27 28

Screen Shot 2022-06-15 at 8 27 52

Please fix it.

Screen Shot 2022-06-15 at 8 28 31

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @FaranoshAmini and @saqibjavaiddev ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job overall implementing the popup window

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: popup.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry',
+    img: './Images/Desktop-version/nature.png',
+    technolgies: ['React', 'Redux', 'HTML', 'CSS', 'JavaScript'],
+    linkToLiveVersion: '#',
+    linkToSource: 'https://github.com/FaranoshAmini',
+    generalInf: ['Microverse- Student', 'Web Developer', 2021],
+  },
+};
+const myWork = document.querySelector('#work');
+function createMobileProjectCard(key) {
```
Comment (topeogunleye) [inline popup.js:34]: - [ ] As suggested by the previous code reviewer, I kindly suggest that your popup follows the design for both desktop and mobile versions from the chosen template in Figma (e.g., font, colours, images, tests, margins).

I can see that the colors for your *Microverse- Student* and *Web Developer • 2021* are wrong. please make use of the colors from your design template

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @FaranoshAmini,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comment.
>- [ ] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Wrong
Screenshot from 2022-06-16 09-12-33
> ### Good Commit Messages:
>- [x]  Add fix for dark mode toggle state
>- [x] Update README file.
>- [x] Add styles to h1.
>
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @FaranoshAmini       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**